[quant][embedding qat] Add eager QAT test for EmbeddingBag+Linear model

b-koopman opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #66560
* #66429
* __->__ #66334

Differential Revision: [D31618283](https://our.internmc.facebook.com/intern/diff/D31618283)

b-koopman committed:
[quant][embedding qat] Add eager QAT test for EmbeddingBag+Linear model

[ghstack-poisoned]

b-koopman committed:
Update on "[quant][embedding qat] Add eager QAT test for EmbeddingBag+Linear model"




[ghstack-poisoned]

b-koopman committed:
Update on "[quant][embedding qat] Add eager QAT test for EmbeddingBag+Linear model"




[ghstack-poisoned]

b-koopman committed:
Update on "[quant][embedding qat] Add eager QAT test for EmbeddingBag+Linear model"




[ghstack-poisoned]

b-koopman committed:
Update on "[quant][embedding qat] Add eager QAT test for EmbeddingBag+Linear model"


Differential Revision: [D31618283](https://our.internmc.facebook.com/intern/diff/D31618283)

[ghstack-poisoned]

HDCharles reviewed (approved):
looks good but I'd make sure to clear that failing xenial test before trying to land or at least figure out if its okay to ignore.

b-koopman committed:
Update on "[quant][embedding qat] Add eager QAT test for EmbeddingBag+Linear model"


Differential Revision: [D31618283](https://our.internmc.facebook.com/intern/diff/D31618283)

[ghstack-poisoned]
